Add alternative view for variables summary

giuscri opened the pull request:
See #9239 . ~**Note that no colors are printed, unlike what's proposed in the issue.**~

```
[0x00000410]> e asm.varsum 
true
[0x00000410]> pdf
            ;-- section..text:
            ;-- eip:
╭ (fcn) entry0 49
│ bp: 0 (vars 0, args 0)
│ sp: 0 (vars 0, args 0)
│ rg: 0 (vars 0, args 0)
...
[0x00000410]> e asm.varsum=1
[0x00000410]> pdf
            ;-- section..text:
            ;-- eip:
╭ (fcn) entry0 49
│ bp: 0 (vars 0, args 0)
│ sp: 0 (vars 0, args 0)
│ rg: 0 (vars 0, args 0)
...
[0x00000410]> e asm.varsum=2
[0x00000410]> pdf
            ;-- section..text:
            ;-- eip:
╭ (fcn) entry0 49
│ vars: 0 0 0
│ args: 0 0 0
...
[0x00000410]> 
```

~[0x565d254d]> e asm.varsum.altern=true~
~[0x565d254d]> pdf|head -n6~
~╭ (fcn) sym.fn 30~
~│ vars: 1 0 0~
~│ args: 1 0 0~
~│           0x565d254d      55             push ebp~
~│           0x565d254e      89e5           mov ebp, esp~
~│           0x565d2550      83ec10         sub esp, 0x10~
~[0x565d254d]>~
~[0x565d254d]>~
~[0x565d254d]>~
~[0x565d254d]> e asm.varsum.altern=false~
~[0x565d254d]> pdf|head -n6~
~╭ (fcn) sym.fn 30~
~│ bp: 2 (vars 1, args 1)~
~│ sp: 0 (vars 0, args 0)~
~│ rg: 0 (vars 0, args 0)~
~│           0x565d254d      55             push ebp~
~│           0x565d254e      89e5           mov ebp, esp~

radare commented:
what about making varsum an int instead of bool and show different format depending on value?

also, there are conflicts in disasm.c can you rebase?

giuscri commented:
Rebased.

@radare, you mean something like `asm.varsum=0` for no summary, `asm.varsum=1` for current variables summary and `asm.varsum=2` for the view proposed in #9306?

Doesn't that have the disadvantage of breaking current radare2rc's?

giuscri commented:
Added colored output.

Removed `asm.varsum.altern`, changed varsum into an integer `(0, 1, 2)`.

Also this should _not_ break .radare2rc's that use a boolean for `asm.varsum`, as `asm.varsum=false|true` will be interpreted respectively as 0 and 1.

radare commented:
Thanks!
